### ISSUE TITLE
"following" modrinth list

### ISSUE THREAD

**Issue by Barmem** [PrismLauncher/PrismLauncher#1702]
### Role

I make a lot of mod packs

### Suggestion

I want to be able to see list of mods that i follow on modrinth

### Benefit

It will allow quicker addition of mods that i often use

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

this one 
![изображение](https://github.com/PrismLauncher/PrismLauncher/assets/70278808/6ddf0309-f870-451d-a4cd-7bcac3bf57d6)

**Comment by josephrooks** [comment]
+1 for this..... just spent so much time favoriting mods I want to go back and look at later, only to not be able to see a list of them... 😐

**Comment by Suitov** [comment]
Yeah... notifications feed is not adequate for this. This feature is pretty basic and I'm surprised it isn't present, but I'd appreciate a lot if it could be added.
